.meta/transforms/dedupe.toml

```diff
@@ -0,0 +1,43 @@
+[transforms.dedupe]
+title = "Dedupe events"
+allow_you_to_description = "prevent duplicate Events from being outputted"
+beta = false
+common = false
+function_category = "filter"
+input_types = ["log"]
+output_types = ["log"]
+requirements = {}
+
+<%= render("_partials/_component_options.toml", type: "transform", name: "dedupe") %>
+
+[transforms.dedupe.options.fields]
+type = "table"
+description = "Options controlling what fields to match against"
+required = true
+
+[transforms.dedupe.options.fields.children.match]
```

Comment:
From this description it's not clear that these are mutually exclusive options, do you think we could add that?